### DISCUSSION TITLE
esp32 wifi antenna switching via software

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#12058] [Hardware & Peripherals]
The pycom wipy3 had the facility to switch from internal pcb antenna to ipex connector for external antenna but all the subsequent esp32 boards seem to favour moving a tiny resistor with a soldering iron to achieve the same result. Does anyone know of an esp32 dev board that offers software switching of the wifi RF path?

**Comment by bidrohini** [comment]
As far as I know, , most ESP32 development boards do not offer built-in software switching of the WiFi RF path between the internal PCB antenna and an external antenna connected via an IPEX connector. The traditional method for switching between antennas on most ESP32 boards involves changing the hardware configuration by moving a resistor or a zero-ohm jumper.
